templates/topic/index.json-ld

```diff
@@ -194,7 +202,7 @@
 		},
 		{
 			"@id": "_:implement_topic_aem",
-			"iteration": "_:iteration_topic_3",
+			"iteration": "_:iteration_topic_4",
```

Comment:
For this one, add an instruction note that mention about the "Contributor" component that are located in the "page details" configuration